testsuites/CBLTester/CBL_Functional_tests/APITests/test_collection.py

```diff
@@ -0,0 +1,264 @@
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        retrived_docId = self.doc_obj.getId(retrived_doc)
+        assert docId == retrived_docId, "Document retrival failed"
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_deleteDocument(self, db_name, collectionName, docId):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
+        created_collection = self.db_obj.createCollection(db, collectionName, scopeName)
+        doc = self.doc_obj.create(docId)
+        self.collection_obj.saveDocument(created_collection, doc)
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        assert self.collection_obj.deleteDocument(created_collection, retrived_doc) == True
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_purgeDocument(self, db_name, collectionName, docId):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
+        created_collection = self.db_obj.createCollection(db, collectionName, scopeName)
+        doc = self.doc_obj.create(docId)
+        self.collection_obj.saveDocument(created_collection, doc)
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        assert self.collection_obj.purgeDocument(created_collection, retrived_doc) == True
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_purgeDocumentById(self, db_name, collectionName, docId):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
+        created_collection = self.db_obj.createCollection(db, collectionName, scopeName)
+        doc = self.doc_obj.create(docId)
+        self.collection_obj.saveDocument(created_collection, doc)
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        retrived_docId = self.doc_obj.getId(retrived_doc)
+        assert self.collection_obj.purgeDocumentById(created_collection, retrived_docId) == True
+
+    @pytest.mark.parametrize("db_name, collection_name, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_documnetExpiration(self, db_name, collection_name, docId):
```

Comment:
Is there a special code for document expiration in the context of scopes and collections? If there isn't, maybe we don't need this test, as (hopefully) document expiration  is tested some plac else?